manifests/cf-manifest/runtime-config/runtime-config-base.yml

```diff
@@ -1,7 +1,4 @@
 ---
-meta:
```

Comment:
I suggest if other things need it they should add it explicitly and describe the need in the commit message.